benchmarks/single_node/config.yaml

```diff
@@ -0,0 +1,39 @@
+cluster_name: distributed-benchmarks
```

Comment:
Why don't we just move it to release/benchmarks folder?

Reply:
I thought one of the goals was to make this envelope very visible and reproducible for end users (whereas the release folder is really targeted at ray's developers).

I don't feel strongly about this though.

Reply:
I think as long as you clearly state how to run/record the result during the release process, I'd fine with this.

Reply:
added docs to the release process